replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+
+logit3logses <- glm(context ~ lgses, data = clean_2a, family = "binomial")
+
+print(wald.test(b = coef(logit3logses), Sigma = vcov(logit3logses), Terms = 2)$result$chi2[1], digit = 3)
+```
+
+> but lower-income participants were more likely than higher-income participants to use trade-off thinking, beta = -0.96, Wald-test chi square (1, N = 237) = 4.69, p < 0.05.
+
+```{r echo = FALSE}
+glm(tradeoff ~ lgses, data = clean_2a, family = "binomial") %>%
+  tidy() %>%
+  filter(term == "lgses") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+logit4logses <- glm(tradeoff ~ lgses, data = clean_2a, family = "binomial")
+
+print(wald.test(b = coef(logit4logses), Sigma = vcov(logit4logses), Terms = 2)$result$chi2[1], digit = 3)
+```
+
+
+### Study 2b
+
+> In study 2b there were 705 participants (mean age = 30.0 years; 290 females, 415 males; median household size = 3 people; median household income = $45,000).  The participants in Study 2c were the same participants as in Study 1d (and analyses were limited to the same participants, but including all participants did not change the results).  Note that participants completed Studies 1d and 2c in a counterbalanced order.  Participants in Studies 2b and 2c responded to the same scenario as did participants in Study 2a, but they stated whether they would travel for the discount.
+
+```{r echo = FALSE}
+
+#reads the data for study 2a - spss file, csv file for demographics, and second csv file for income
+suppressMessages(suppressWarnings(raw_2b_spss <- read.spss("framing/study_2b.sav", to.data.frame = TRUE)))
+suppressMessages(suppressWarnings(raw_2b_csv <- read.csv("study_2b_demog.csv", header = TRUE)))
+
+#average age for Study 2b
+raw_2b_csv %>%
+  filter(age != "NA") %>%
+  summarise(mean.age = mean(age)) %>%
+  round(digits = 1)
+
+#calculate sex breakdown for Study 2b
+raw_2b_csv %>%
+  group_by(gender) %>%
+  summarise(sex_breakdown = n())
+
+#calculate total sample size for Study 2b
+raw_2b_csv %>%
+  summarise(sample.size = n())
+
+#calculate median household income and size for Study 2b
+raw_2b_spss %>%
+  summarise(median.household.income = median(inc2), median.household.size = median(house))
+```
+
+
+***Results***
+-------
+
+> In Study 2b, 1 participant was excluded for missing responses, and 2 were excluded as outliers on income.
+
+```{r echo = FALSE}
+clean_2b <- raw_2b_spss %>%
+  filter(filter_. != "Not Selected", go != "NA")
+
+clean_2b %>%
+  summarise(cleaned.sample.size = n())
+```
+
+> The analyses conducted were based on a median split on income.  Confirming earlier findings, these analyses revealed that higher-income participants were more willing to travel when the discount was proportionally larger (i.e., the tablet cost less): Specifically, 86%, 75%, and 58% would travel when the tablet cost $300, $500, and $1,000, respectively.
+
+```{r echo = FALSE}
+clean_2b %>%
+  filter(split == 1, cond == 0) %>%
```

Comment:
rather than continual filtering, consider creating a dataset with the filtered data that you'll use in the proceeding analyses - that way, any edits (or transferred code from a different project) only need to be customized once.